calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java

```diff
@@ -4041,6 +4041,10 @@ public String buildIcalExportPanelContext(VelocityPortlet portlet, Context conte
 		boolean exportEnabled = CalendarService.getExportEnabled(calId);
 		context.put("enable_export", String.valueOf(exportEnabled) );
 
+		String icalInfoArr[] = {String.valueOf(ServerConfigurationService.getInt("calendar.export.previous.months",6)),String.valueOf(ServerConfigurationService.getInt("calendar.export.next.months",6)),rb.getFormattedMessage("ical.info.public")};
```

Comment:
Also can you wrap you're lines so they aren't as long. Sort of unwritten rule is over 120 characters is too much (although it's not hard and fast).